crates/solidity/outputs/npm/package/src/kinds/index.ts

```diff
@@ -5,3 +5,6 @@ export type RuleKind = generated.kinds.RuleKind;
 
 export const TokenKind = generated.kinds.TokenKind;
 export type TokenKind = generated.kinds.TokenKind;
+
```

Comment:
I suggest adding a TS unit test that uses this new API to `crates/solidity/outputs/npm/tests/`

Reply:
Done!